jax/_src/distributed.py

```diff
@@ -70,6 +74,18 @@ def initialize(self,
 
     self.process_id = process_id
 
+    # Emit a warning about PROXY variables if they are in the user's env:
+    proxy_vars = [ key for key in os.environ.keys() if 'proxy' in key.lower()]
```

Comment:
could it be `_proxy` to be narrower?

Reply:
Sure, done, good suggestion.  This deliberately only emits a warning, though that can get obnoxious if firing superficially at 10000 ranks.